dist/streams.js

```diff
@@ -169,10 +170,22 @@ class MetacomWritable extends EventEmitter {
 
   write(data) {
     const chunk = MetacomChunk.encode(this.streamId, data);
-    this.transport.send(chunk);
+    this.transport.send(chunk, (err) => {
```

Comment:
If you need error handler from socket, it would be better to mage `transport.send` function async and use await instead of accessing socket callback.